packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -59,6 +56,47 @@ describe('HasMany relation', () => {
     expect(persisted.toObject()).to.deepEqual(order.toObject());
   });
 
+  it('can patch many instances', async () => {
+    await givenCustomerOrder({description: 'order 1', isDelivered: false});
+    await givenCustomerOrder({description: 'order 2', isDelivered: false});
+    const patchObject = {isDelivered: true};
+    const arePatched = await customerOrders.patch(patchObject);
+    expect(arePatched).to.equal(2);
+    const patchedData = _.map(await customerOrders.find(), d =>
+      _.pick(d, ['customerId', 'description', 'isDelivered']),
+    );
+    expect(patchedData).to.eql([
+      {
+        customerId: existingCustomerId,
+        description: 'order 1',
+        isDelivered: true,
+      },
+      {
+        customerId: existingCustomerId,
+        description: 'order 2',
+        isDelivered: true,
+      },
+    ]);
+  });
+
+  it('throws error when query tries to change the foreignKey', async () => {
+    await expect(
+      customerOrders.patch({customerId: existingCustomerId + 1}),
+    ).to.be.rejectedWith(/Property "customerId" cannot be changed!/);
+  });
+
+  it('can delete many instances', async () => {
+    await givenCustomerOrder({description: 'order 1'});
+    await givenCustomerOrder({description: 'order 2'});
+    const deletedOrders = await customerOrders.delete();
+    expect(deletedOrders).to.equal(2);
+    const relatedOrders = await customerOrders.find();
+    expect(relatedOrders).to.be.empty();
```

Comment:
Should we also assert that the contrained delete does not delete data that it's not supposed to delete?